pkg/drivers/kic/kic.go

```diff
@@ -305,6 +305,11 @@ func (d *Driver) Restart() error {
 // Start an already created kic container
 func (d *Driver) Start() error {
 	if err := oci.StartContainer(d.NodeConfig.OCIBinary, d.MachineName); err != nil {
+		oci.PrintPostPortem(d.OCIBinary, d.MachineName)
+		_, err := oci.DaemonInfo(d.OCIBinary)
+		if err != nil {
+			return errors.Wrapf(oci.ErrDaemonInfo, "container name %q", d.MachineName)
```

Comment:
The wrapped error should mention something about the daemon info call.

Reply:
done